### PR TITLE
Sprint12 04 a45587 base port to centos6

### PR DESCRIPTION
**Pull request by caryp**
Add CentOS 6 support for Base and LB

### CODE REVIEW

File: cookbooks/rightscale/attributes/monitoring.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      set_unless[:rightscale][:collectd_remove_existing] = true
+    when /6\..+/
+      set_unless[:rightscale][:collectd_packages_version] = "4.10.7-1.el6"
+    end
+  else
+    raise "Unrecognized distro #{node[:platform]} for monitoring attributes , exiting "
+end
+
+default[:rightscale][:plugin_list] = ""
+default[:rightscale][:plugin_list_ary] = [
```
Comment (ekargard) [inline cookbooks/rightscale/attributes/monitoring.rb]: should this be 'array' and not 'ary'

Comment (caryp) [inline cookbooks/rightscale/attributes/monitoring.rb]: probably, but that is outside the scope of this change.  These values were just moved from the default attributes file to untangle them from the rest of the attributes.

File: cookbooks/rightscale/attributes/monitoring.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    set_unless[:rightscale][:collectd_packages] = ["collectd"]
+    set_unless[:rightscale][:collectd_config] = "/etc/collectd.conf"
+    set_unless[:rightscale][:collectd_plugin_dir] = "/etc/collectd.d"
+    case platform_version
+    when /5\..+/
+      set_unless[:rightscale][:collectd_packages_version] = "4.10.0-4.el5"
+      set_unless[:rightscale][:collectd_remove_existing] = true
+    when /6\..+/
+      set_unless[:rightscale][:collectd_packages_version] = "4.10.7-1.el6"
+    end
```
Comment (douglaswth) [inline cookbooks/rightscale/attributes/monitoring.rb]: I think that this should explicitly match at the beginning of the version string.

``` ruby
case platform_version
when /^5\..+/
  ...
when /^6\..+/
  ...
end
```

Otherwise it is possible something could possibly match when it shouldn't.

File: cookbooks/rightscale/attributes/monitoring.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  else
+    raise "Unrecognized distro #{node[:platform]} for monitoring attributes , exiting "
+end
```
Comment (douglaswth) [inline cookbooks/rightscale/attributes/monitoring.rb:30]: This case statement should not have extra indentation. Please see [Code Style Conventions](https://wookiee.rightscale.com/display/rightscale/ServerTemplate+Development+Guide#ServerTemplateDevelopmentGuide-CodeStyleConventions) for our coding style.
